scss/_patterns_tooltips.scss

```diff
@@ -1,5 +1,7 @@
 @import 'settings';
 $triangle-height: $sp-unit;
+$bottom-border-radius: 100% 100% 0% 0%;
```

Comment:
I know I suggested such names, but the may be confusing without context.
It sounds like it's related to bottom border radius of some element, while it is a border radius of bottom positioned tooltip.

I would at least prefix the variables with tooltip `$tooltip-bottom-border-radius` (to make sure it doesn't collide with some other variable in sass).
And maybe add a comment above those to give a bit more context. Something like:
`// these are the values of second border radius used for hover overlays in bottom and top aligned tooltips` ?